kubectl/step12.md

```diff
@@ -1,8 +1,8 @@
-If you want to sort the information output, this flag is helpful:
+If you want to sort the output of queries, you can use the `--sort-by` flag:
 
 `kubectl get pod --sort-by=.` by some data point. 
 
 In this case, let's sort by name
```

Comment:
It might be nice to show an example where it's more clear about the sorting taking place. The pod names are very similar and it will pretty much come down to what the dynamically generated aspects of the name are.